infrastructure/main.tf

```diff
@@ -70,6 +70,18 @@ data "vault_generic_secret" "oauth_client_secret" {
   path = "secret/${var.vault_section}/ccidam/idam-api/oauth2/client-secrets/cmc-citizen"
 }
 
+module "quartz-store-db" {
+  source = "git@github.com:hmcts/moj-module-postgres?ref=cnp-449-tactical"
+  product = "quartz-postgres-db"
```

Comment:
please user:
`${var.product}-scheduler`

don't need postgres-db in there, and rename the module to be scheduler